packages/grafana-ui/src/components/TextArea/TextArea.tsx

```diff
@@ -24,6 +24,7 @@ const getTextAreaStyle = stylesFactory((theme: GrafanaTheme2, invalid = false) =
       sharedInputStyle(theme),
       getFocusStyle(theme),
       css`
+        display: block;
```

Comment:
It changes `inline-block` to `block` - are we sure it won't affect any existing components? If not, would it be better to change CSS for our use case only and create a separate issue just to change `display: block`?